tools/gather-sysinfo/gather-sysinfo.go

```diff
@@ -180,5 +180,7 @@ func kniExpectedCloneContent() []string {
 		"/sys/class/dmi/id/product_name",
 		"/sys/class/dmi/id/product_sku",
 		"/sys/class/dmi/id/product_version",
+		// KNI-specific CPU info
+		"/sys/devices/system/cpu/offline",
```

Comment:
Inhttps://github.com/openshift-kni/performance-addon-operators/pull/878#discussion_r855783055 I meant line 174, I should have added the line number in my previous comment. No big deal, not worth resubmitting *for this reason*.

Reply:
Right sorry, miss the comment was there